Mathlib/Control/LawfulFix.lean

```diff
@@ -182,15 +189,16 @@ def toUnitMono (f : Part α →o Part α) : (Unit → Part α) →o Unit → Par
   toFun x u := f (x u)
   monotone' x y (h : x ≤ y) u := f.monotone <| h u
 
-theorem to_unit_cont (f : Part α →o Part α) (hc : Continuous f) : Continuous (toUnitMono f)
-  | _ => by
```

Comment:
Please restore this lemma, but tagged `@[deprecated ωScottContinuous_toUnitMono (since := "2024-08-26")]`